pkg/installer/directpv.min.io_directpvvolumes.yaml

```diff
@@ -0,0 +1,149 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
this file looks unrelated, can you remove?

Reply:
it was present in the file before https://github.com/minio/directpv/blob/master/pkg/admin/installer/directpv.min.io_directpvvolumes.yaml

Reply:
these files are not present in the existing source. can you remove these?